### PR TITLE
Makefile: ignore unit test coverage for generated files

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6140)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Review by jrajahalme — APPROVED** [review]

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -85,16 +85,19 @@ tests: force
 
 unit-tests: start-kvstores
 	$(QUIET) $(MAKE) -C daemon/ check-bindata
-	$(QUIET) echo "mode: count" > coverage-all.out
+	$(QUIET) echo "mode: count" > coverage-all-tmp.out
 	$(QUIET) echo "mode: count" > coverage.out
 	$(QUIET)$(foreach pkg,$(TESTPKGS),\
 		$(GO) test $(pkg) $(GOTEST_BASE) $(GOTEST_COVER_OPTS) || exit 1; \
```
Comment (ianvernon) [inline Makefile:91]: Is there a way we can remove `github.com/cilium/cilium/api/v1` and `github.com/cilium/cilium/pkg/k8s/client` from `TESTPKGS`?

**Pull request merged**